### PR TITLE
www.realtek.com mobile site fix

### PR DESCRIPTION
**Pull request by sprinkledcheese**
Inverted header colour, also made "search", "globe" button transparent, and "MORE" button dark.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4025,6 +4025,28 @@ IGNORE INLINE STYLE
 
 ================================
 
+www.realtek.com
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4028]: Remove www. to avoid any compatible issues.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -4025,6 +4025,28 @@ IGNORE INLINE STYLE
...
+
+CSS
+#t3-header
+{
+    background: none !important;
+}
+#t3-header > div > div.col-xs-3.visible-xs-block > button.search-toggle
+{
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4036]: `{` on the same line as the selector only different selectors on seperate lines.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:4028]: "realtek.com" doesn't exist.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:4028]: Yes but some files can be loaded from sub-domains etc. We only use www. for rare cases where it's needed.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config]: Done.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:4028]: ok.

**Pull request merged**